.github/workflows/discord-posts.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           sudo apt-get install -y apt-transport-https
           echo "deb [arch=amd64 signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list
           sudo apt-get update
-          sudo apt-get install gh
+          sudo apt-get install -y gh
```

Comment:
This should probably be `apt` instead of `apt-get` but it still will technically work in 22.04 and 24.04 to use `apt-get` in compatibility mode. It would be more future-proof to just use `apt` here though.

Reply:
completely agreed, but at this hour I just want to merge. we have other places that still use apt-get as well.